### ISSUE TITLE
General Issue: Post-Release Tidying, Cleaning and Power Washing

### ISSUE THREAD

**Issue by bms63** [pharmaverse/admiral#1944]
### Background Information

Various tasks to be done to make next release smoother and keep admiral clean and tidy.  To be completed right after admiral release.

### Definition of Done

- [ ] Remove all unused branches in `admiral`, `admiral.test`, `admiraldev` 
- [ ] Close remaining PR-linked issues not close by merged to `main`
- [ ] Remove contributors who have not participated in `admiral`, `admiral.test`, `admiraldev` for over a year
- [ ] Create a scope issue for next release
- [ ] Use scope issue to drive Priority Column in Project Board.
- [ ] Create a deprecation issue for next release.
- [ ] Create a stand-in section `News.md` for next release (place all relevant sections)
- [ ] Up-version `DESCRIPTION` file to next release  
- [ ] Check that pharmaverse/admiral team is up to date with current members
- [ ] Update `patch` branch with latest release

**Comment by bms63** [comment]
@pharmaverse/admiral  Is there anyone we should add to the core team?  That way wasy to tag folks in the issue?  Makes it easier to get a discussion going

Please see discussion on scope for `0.12.0` https://github.com/pharmaverse/admiral/discussions/1970

**Comment by bms63** [comment]
@kaz462 while working on `admiraldata` from `admiraltemplate` - have you noticed any updates or explanations that are needed?

**Comment by kaz462** [comment]
> @kaz462 while working on `admiraldata` from `admiraltemplate` - have you noticed any updates or explanations that are needed?

Yes - for  `admiraltemplate` [`README`](https://github.com/pharmaverse/admiraltemplate#quick-start-guide-for-template), can we add `.Rbuildignore` to the following part? 

![image](https://github.com/pharmaverse/admiraltemplate/assets/47867131/f95d4ef8-b4be-46bc-8a5d-f809829529fd)

**Comment by bms63** [comment]
I think we are good on this one @pharmaverse/admiral 

Our scope issue/discussion is still pending, but that one will be a while.

> > @kaz462 while working on `admiraldata` from `admiraltemplate` - have you noticed any updates or explanations that are needed?
> 
> Yes - for `admiraltemplate` [`README`](https://github.com/pharmaverse/admiraltemplate#quick-start-guide-for-template), can we add `.Rbuildignore` to the following part?
> 
> ![image](https://user-images.githubusercontent.com/47867131/248937890-f95d4ef8-b4be-46bc-8a5d-f809829529fd.png)

@kaz462 issue made in admiraltemplate
